packages/tocco-ui/src/Select/LoadingIndicator.js

```diff
@@ -2,6 +2,6 @@ import React from 'react'
 
 import IconTocco from '../IconTocco'
 
-const LoadingIndicator = () => <IconTocco size="1.2em"/>
+const LoadingIndicator = () => <span><IconTocco size="1.8rem"/></span>
```

Comment:
Why is the span needed?

Reply:
I've introduced a new wrapping element for buttons to attach events and props (see ClearIndicator.js and IndicatorsContainer.js). LoadingIndicator.js and DropdownIndicator.js does contain that wrapping span as well in order to use the very same css.